<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>863</width>
    <height>684</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="tab_ConnectionSettings">
         <attribute name="title">
          <string>Настройка</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QGroupBox" name="groupBox">
            <property name="title">
             <string>Камеры</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <layout class="QGridLayout" name="gridLayout_3">
               <item row="0" column="0">
                <widget class="QLabel" name="label_camPort">
                 <property name="text">
                  <string>Порт</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1" colspan="2">
                <widget class="QLineEdit" name="lineEdit_camsPort"/>
               </item>
               <item row="1" column="0" colspan="2">
                <widget class="QCheckBox" name="checkBox_cam1">
                 <property name="text">
                  <string>Камера1 IP</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="2">
                <widget class="QLineEdit" name="lineEdit_cam1Ip"/>
               </item>
               <item row="2" column="0" colspan="2">
                <widget class="QCheckBox" name="checkBox_cam2">
                 <property name="text">
                  <string>Камера2 IP</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="2">
                <widget class="QLineEdit" name="lineEdit_cam2Ip"/>
               </item>
               <item row="3" column="0" colspan="2">
                <widget class="QCheckBox" name="checkBox_cam3">
                 <property name="text">
                  <string>Камера3 IP</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="2">
                <widget class="QLineEdit" name="lineEdit_cam3Ip"/>
               </item>
               <item row="4" column="0" colspan="2">
                <widget class="QCheckBox" name="checkBox_cam4">
                 <property name="text">
                  <string>Камера4 IP</string>
                 </property>
                </widget>
               </item>
               <item row="4" column="2">
                <widget class="QLineEdit" name="lineEdit_cam4Ip"/>
               </item>
               <item row="5" column="0" colspan="2">
                <widget class="QCheckBox" name="checkBox_cam5">
                 <property name="text">
                  <string>Камера5 IP</string>
                 </property>
                </widget>
               </item>
               <item row="5" column="2">
                <widget class="QLineEdit" name="lineEdit_cam5Ip"/>
               </item>
               <item row="6" column="0" colspan="2">
                <widget class="QCheckBox" name="checkBox_cam6">
                 <property name="text">
                  <string>Камера6 IP</string>
                 </property>
                </widget>
               </item>
               <item row="6" column="2">
                <widget class="QLineEdit" name="lineEdit_cam6Ip"/>
               </item>
              </layout>
             </item>
             <item>
              <spacer name="verticalSpacer_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_cameraTerminal">
               <property name="text">
                <string>Открыть терминал</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_2">
            <property name="title">
             <string>Стенд</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <layout class="QGridLayout" name="gridLayout_4">
               <item row="1" column="2">
                <widget class="QLineEdit" name="lineEdit_standIp"/>
               </item>
               <item row="0" column="2">
                <widget class="QLineEdit" name="lineEdit_standPort"/>
               </item>
               <item row="0" column="1">
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>Порт</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>IP</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>382</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_standTerminal">
               <property name="text">
                <string>Открыть терминал</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_CommonTest">
         <attribute name="title">
          <string>Общее</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QFrame" name="frame">
            <layout class="QGridLayout" name="gridLayout">
             <property name="spacing">
              <number>7</number>
             </property>
             <item row="3" column="1">
              <widget class="QPushButton" name="pushButton_cam5">
               <property name="text">
                <string>Камера 5</string>
               </property>
              </widget>
             </item>
             <item row="3" column="2">
              <widget class="QPushButton" name="pushButton_cam6">
               <property name="text">
                <string>Камера 6</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QFrame" name="frame_video_cam1">
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QPushButton" name="pushButton_cam1">
               <property name="text">
                <string>Камера 1</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QFrame" name="frame_video_cam2">
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QPushButton" name="pushButton_cam2">
               <property name="text">
                <string>Камера 2</string>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QPushButton" name="pushButton_testAll">
               <property name="text">
                <string>Проверить всё</string>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QFrame" name="frame_video_cam3">
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QPushButton" name="pushButton_cam4">
               <property name="text">
                <string>Камера 4</string>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QPushButton" name="pushButton_cam3">
               <property name="text">
                <string>Камера 3</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QFrame" name="frame_video_cam4">
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QFrame" name="frame_video_cam5">
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
              </widget>
             </item>
             <item row="2" column="2">
              <widget class="QFrame" name="frame_video_cam6">
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_SingleTest">
         <attribute name="title">
          <string>Подробнее</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <item>
               <layout class="QFormLayout" name="formLayout">
                <item row="0" column="0">
                 <widget class="QLabel" name="label_4">
                  <property name="text">
                   <string>Камера</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QComboBox" name="comboBox_cams_2"/>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="label_5">
                  <property name="text">
                   <string>Файл теста</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QComboBox" name="comboBox_settingsFiles"/>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QTextEdit" name="textEdit_testSettings"/>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_saveSettings">
                <property name="text">
                 <string>Сохранить</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_7">
              <item>
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>Результат тестирования</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QTextEdit" name="textEdit_testResults"/>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_testCam">
                <property name="text">
                 <string>Тестировать</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_Misc">
         <attribute name="title">
          <string>Разное</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QFrame" name="frame_2">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="7" column="0">
              <widget class="QLabel" name="label_I2C_name">
               <property name="text">
                <string>I2C:</string>
               </property>
              </widget>
             </item>
             <item row="3" column="2">
              <widget class="QLineEdit" name="lineEdit_motor_zoom_threshold"/>
             </item>
             <item row="3" column="3">
              <widget class="QPushButton" name="pushButton_motor_zoom">
               <property name="text">
                <string>Проверить</string>
               </property>
              </widget>
             </item>
             <item row="2" column="3">
              <widget class="QPushButton" name="pushButton_motor_focus">
               <property name="text">
                <string>Проверить</string>
               </property>
              </widget>
             </item>
             <item row="10" column="3">
              <widget class="QPushButton" name="pushButton_LED1">
               <property name="text">
                <string>Проверить</string>
               </property>
              </widget>
             </item>
             <item row="4" column="3">
              <widget class="QPushButton" name="pushButton_IRC">
               <property name="text">
                <string>Проверить</string>
               </property>
              </widget>
             </item>
             <item row="6" column="0">
              <widget class="QLabel" name="label_IR_name">
               <property name="text">
                <string>ИК освещение:</string>
               </property>
              </widget>
             </item>
             <item row="6" column="2">
              <widget class="QLabel" name="label_IR_DIM_ADC3">
               <property name="text">
                <string>0</string>
               </property>
              </widget>
             </item>
             <item row="10" column="1">
              <widget class="QLabel" name="label_LED1">
               <property name="text">
                <string>0</string>
               </property>
              </widget>
             </item>
             <item row="6" column="3">
              <widget class="QPushButton" name="pushButton_IR">
               <property name="text">
                <string>Проверить</string>
               </property>
              </widget>
             </item>
             <item row="12" column="0">
              <widget class="QLabel" name="label_current_name">
               <property name="text">
                <string>Ток питания:</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_motor_zoom_name">
               <property name="text">
                <string>Мотор зума:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Камера</string>
               </property>
              </widget>
             </item>
             <item row="5" column="1">
              <widget class="QLabel" name="label_12">
               <property name="text">
                <string>ШИМ:</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_motor_focus_name">
               <property name="text">
                <string>Мотор фокуса:</string>
               </property>
              </widget>
             </item>
             <item row="6" column="1">
              <widget class="QLabel" name="label_IR_CDS_PWM">
               <property name="text">
                <string>0</string>
               </property>
              </widget>
             </item>
             <item row="8" column="0">
              <widget class="QLabel" name="label_IRIS_name">
               <property name="text">
                <string>Диафрагма IRIS:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QComboBox" name="comboBox_cams"/>
             </item>
             <item row="8" column="2">
              <widget class="QLineEdit" name="lineEdit_IRIS_threshold"/>
             </item>
             <item row="10" column="0">
              <widget class="QLabel" name="label_LED1_name">
               <property name="text">
                <string>LED1 Ethernet:</string>
               </property>
              </widget>
             </item>
             <item row="8" column="3">
              <widget class="QPushButton" name="pushButton_IRIS">
               <property name="text">
                <string>Проверить</string>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="label_IRC_name">
               <property name="text">
                <string>ИК шторка:</string>
               </property>
              </widget>
             </item>
             <item row="9" column="3">
              <widget class="QPushButton" name="pushButton_UART">
               <property name="text">
                <string>Проверить</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QLabel" name="label_motor_zoom">
               <property name="text">
                <string>0</string>
               </property>
              </widget>
             </item>
             <item row="4" column="2">
              <widget class="QLabel" name="label_IRC_GPIO2">
               <property name="text">
                <string>0</string>
               </property>
              </widget>
             </item>
             <item row="7" column="2">
              <widget class="QLabel" name="label_15">
               <property name="text">
                <string>message from I2C</string>
               </property>
              </widget>
             </item>
             <item row="9" column="1">
              <widget class="QLineEdit" name="lineEdit_UART_message">
               <property name="text">
                <string>message to UART</string>
               </property>
              </widget>
             </item>
             <item row="9" column="2">
              <widget class="QLabel" name="label_19">
               <property name="text">
                <string>message from UART</string>
               </property>
              </widget>
             </item>
             <item row="7" column="1">
              <widget class="QLineEdit" name="lineEdit_I2C_message">
               <property name="text">
                <string>message to I2C</string>
               </property>
              </widget>
             </item>
             <item row="11" column="3">
              <widget class="QPushButton" name="pushButton_RESET">
               <property name="text">
                <string>Проверить</string>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QLabel" name="label_IRC_GPIO1">
               <property name="text">
                <string>0</string>
               </property>
              </widget>
             </item>
             <item row="11" column="0">
              <widget class="QLabel" name="label_RESET_name">
               <property name="text">
                <string>Сигнал RESET:</string>
               </property>
              </widget>
             </item>
             <item row="7" column="3">
              <widget class="QPushButton" name="pushButton_I2C">
               <property name="text">
                <string>Проверить</string>
               </property>
              </widget>
             </item>
             <item row="8" column="1">
              <widget class="QLabel" name="label_IRIS">
               <property name="text">
                <string>0</string>
               </property>
              </widget>
             </item>
             <item row="2" column="2">
              <widget class="QLineEdit" name="lineEdit_motor_focus_threshold"/>
             </item>
             <item row="2" column="1">
              <widget class="QLabel" name="label_motor_focus">
               <property name="text">
                <string>0</string>
               </property>
              </widget>
             </item>
             <item row="9" column="0">
              <widget class="QLabel" name="label_UART_name">
               <property name="text">
                <string>UART:</string>
               </property>
              </widget>
             </item>
             <item row="11" column="1">
              <widget class="QLabel" name="label_RESET">
               <property name="text">
                <string>0</string>
               </property>
              </widget>
             </item>
             <item row="5" column="2">
              <widget class="QLabel" name="label_13">
               <property name="text">
                <string>АЦП:</string>
               </property>
              </widget>
             </item>
             <item row="12" column="1">
              <widget class="QLabel" name="label_current">
               <property name="text">
                <string>0</string>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QPushButton" name="pushButton_showLogo">
               <property name="text">
                <string>ShowLogo</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>863</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
